content/docs/user-guide/assets/asset-processor/settings.md

```diff
@@ -0,0 +1,68 @@
+---
+linkTitle: Asset Pipeline Settings
+title: Asset Pipeline Settings
+description: Commandline Settings for AssetProcessor and AssetBuilder
+weight: 900
+toc: true
+---
+
+The asset pipelines  exposes a number of commandline settings that can be used when manually launching.
+
+## Asset Processor settings
+
+When manually launching Asset Processor or Asset Processor Batch you can set the following general properties:
+
+| Setting | Description                                                         |
+|---------|---------------------------------------------------------------------|
+| project-name | Name of the current project.                                        |
+| project-cache-path | Full path to the project cache folder.                              |
+| project-path | Supplies the path to the project that the AP should use (required). | 
+| engine-path | Supplies the engine path to the engine.                             |
+
+You can also control the behavior of Asset Processor or Asset Processor Batch using these settings:
+
+| Setting                         | Description                                                                                                                                                                                         |
+|---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| waitOnLaunch                    | Briefly pauses Asset Processor during initializiation. Useful if you want to attach a debugger.                                                                                                     |
+| zeroAnalysisMode                | Enables using file modification time when examining source assets for processing.                                                                                                                   | 
+| enableQueryLogging              | Enables logging database queries.                                                                                                                                                                   |
+| dependencyScanPattern (dsp)     | Scans assets that match the given pattern for missing product dependencies.                                                                                                                         |
+| fileDependencyScanPattern (fsp) | Used with `dependencyScanPattern` to farther filter the scan                                                                                                                                        |
```

Comment:
nitpick: You're using a period at the end of the other descriptions, but not here

Reply:
Will fix